### PR TITLE
chore: remove py dependency

### PR DESCRIPTION
**Pull request by sandre35**
This fixes issue: https://github.com/pytest-dev/pytest-selenium/issues/313

### CODE REVIEW

File: src/pytest_selenium/drivers/crossbrowsertesting.py [hunk 1]
```diff
@@ -113,13 +112,9 @@ def driver_kwargs(request, test, capabilities, **kwargs):
...
-        "controls": "",
-        "poster": video.get("image"),
-        "play-pause-on-click": "",
-        "style": "border:1px solid #e6e6e6; float:right; height:240px; "
-        "margin-left:5px; overflow:hidden; width:320px",
-    }
-    source_attrs = {"src": video.get("video"), "type": "video/mp4"}
-    return str(html.video(html.source(**source_attrs), **video_attrs))
+    return (
+        f'<video controls="" play-pause-on-click="" poster="{video.get("image")}" style="border:1px solid #e6e6e6;'
```
Comment (RonnyPfannschmidt) [inline src/pytest_selenium/drivers/crossbrowsertesting.py:116]: This is missing safe quoting off the inputs

**Comment by BeyondEvil** [comment]
Looks good!

Please also add an entry to the changelog. 🙏

**Comment by BeyondEvil** [comment]
Please also add a shoutout to the person reporting the issue and to yourself for fixing it. 😊

**Comment by BeyondEvil** [comment]
Not sure why the tests are failing. Can you take a look?

**Comment by sandre35** [comment]
@BeyondEvil After having a look at failing tests, the cause is the release of selenium package. Changes are linked to this issue https://github.com/pytest-dev/pytest-selenium/issues/283

Can we merge this PR and proceed to the 4.0.2 release ?

**Pull request closed without merging**